lib/active_metrics/connection_adapters/miq_postgres_adapter.rb

```diff
@@ -0,0 +1,86 @@
+require 'active_metrics/connection_adapters/abstract_adapter'
+
+module ActiveMetrics
+  module ConnectionAdapters
+    class MiqPostgresAdapter < AbstractAdapter
+      include Vmdb::Logging
+
+      def self.create_connection(_config)
+        ActiveRecord::Base.connection
+      end
+
+      def write_multiple(*metrics)
+        metrics.flatten!
+
+        flatten_metrics(metrics).each do |interval_name, by_resource|
+          by_resource.each do |resource, by_timestamp|
+            start_time = by_timestamp.keys.min
+            end_time   = by_timestamp.keys.max
+            data       = by_timestamp.values
+            write_rows(interval_name, resource, start_time, end_time, data)
+          end
+        end
+
+        metrics
+      end
+
+      private
+
+      def flatten_metrics(metrics)
+        {}.tap do |index|
+          metrics.each do |m|
+            interval_name = m.fetch_path(:tags, :capture_interval_name)
+            resource = m[:resource] || m.fetch_path(:tags, :resource_type).safe_constantize.find(m.fetch_path(:tags, :resource_id))
```

Comment:
Interesting...I didn't expect the resource_type to be in a tag.  I thought for the other adapters, resource_type and resource_id were top-level keys.